Template/Pages/About Us.html

```diff
@@ -1,6 +1,6 @@
-<!--
- Title: About Us
- -->
+<laccPage
+ title="About Us"
```

Comment:
There is no separate metadata comment at the top anymore. Those things are defined on the root element as attributes.

When you generate, it will complain because each page now needs a `description` and `keywords` in addition to the `title` it had before.